clang/test/SemaSYCL/intel-fpga-device-global.cpp

```diff
@@ -0,0 +1,44 @@
+// RUN: %clang_cc1 -fsycl-is-device -std=c++17 -sycl-std=2020 -verify %s
+#include "Inputs/sycl.hpp"
+
+// Tests that [[intel::numbanks()]] can be applied to device_global variables.
+using namespace sycl::ext::oneapi;
+
+[[intel::numbanks(2)]] device_global<int> dev_glob; // OK
```

Comment:
Please add a test case that uses const - something like:
[[intel::numbanks(4)]] const device_global<int> constdev_glob;

Reply:
Done